lldb-eval/eval.h

```diff
@@ -79,8 +93,11 @@ class Interpreter : Visitor {
   // types, create objects, perform casts, etc.
   lldb::SBTarget target_;
 
+  // Flow analysis contains the evaluation information about the current code
+  // branch. It allows to eliminate some "pair" operations (e.g. `&*`).
+  std::vector<FlowAnalysis*> flow_analysis_stack_;
```

Comment:
Instead of using an explicit stack, you could just chain the on-stack flow analysis objects in a linked list (each flow analysis object would have a parent pointer).

Reply:
The problem is that flow analysis objects are not always created (which may be a problem on it's own), so this chaining will be a bit tricky.